src/_redirects

```diff
@@ -0,0 +1,2 @@
+# Allow client-side routing
+/*   /index.html   200
```

Comment:
provide "redirects" to Netlify to give a `200` (success) for ALL routes, and to use `index.html`—at that point, react-router will kicks in and handle routing

client-side routing ftw!